app/src/main/java/com/nextcloud/talk/controllers/CallController.java

```diff
@@ -349,6 +349,8 @@ private void createCameraEnumerator() {
     protected void onViewBound(@NonNull View view) {
         super.onViewBound(view);
 
+        DisplayUtils.applyColorToNavigationBar(getActivity().getWindow(), Color.BLACK);
```

Comment:
looking at some screenshots it might me `#121212` which is the default dark theme background color and might be the one used for the screen too.

Reply:
Ich checkd, atm the background of the call screen is `#111111` or better `@color/grey950`